res/skins/QMLDemo/Deck.qml

```diff
@@ -369,4 +370,15 @@ Item {
 
     }
 
+    DropArea {
+        anchors.fill: parent
+        onDropped: {
+            if (drop.hasUrls) {
+                let url = drop.urls[0];
+                console.log("Dropped URL '" + url + "' on deck " + root.group);
+                root.deckPlayer.loadTrackFromLocationUrl(url);
+            }
```

Comment:
idk if thats good etiquette, but I'd consider setting `drop.accepted = false` in an else case...